docker-compose.yml

```diff
@@ -1,17 +1,20 @@
-web:
-  image: colinxfleming/dcaf_case_management
-  command: rails s -p 3000 -b 0.0.0.0
-  volumes:
-    - .:/usr/src/app
-  ports:
-    - "3000:3000"
-  links:
-    - db
-  environment:
-    mongohost: db
-  env_file:
-    - web-variables.env
-db:
-  image: mongo
-  ports:
-    - "27017:27017"
+version: '3.0'
```

Comment:
I would back off on this until the CircleCI stuff is available across the board. You can build the image (probably worth adding this to the docs) with `docker build -t colinxfleming/dcaf_case_management .` and have it used by docker-compose (or at least this works for me when I need to test things).